code/game/objects/items/control_wand.dm

```diff
@@ -55,8 +58,6 @@
 				else
 					D.emergency = 1
 				D.update_icon()
-	else
```

Comment:
You didn't need to remove the entire thing, just the `if` statement. Pretty sure without the `else` and the `to_chat` below, it won't tell you when you don't have the access required to open a door.